docs/sources/datasources/loki/_index.md

```diff
@@ -60,6 +60,7 @@ datasources:
     access: proxy
     url: http://localhost:3100
     jsonData:
+      timeout: 60
```

Comment:
@JStickler Do you know who on your Loki team could provide a technical review?

Reply:
I would ping @stringbean or @owen-d .  But having searched the code, I'm not sure this setting exists?
I don't see it in either of these files:
https://github.com/grafana/grafana/blob/0f6380b76a6114bdd0454f9b5a529746a5166bf3/pkg/services/datasources/models.go#L42
https://github.com/grafana/grafana/blob/0f6380b76a6114bdd0454f9b5a529746a5166bf3/pkg/api/dtos/datasource.go#L11